src/Alpha_complex/include/gudhi/Incremental_delaunay.h

```diff
@@ -0,0 +1,137 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// As an exception, if you have a commercial license for the CGAL package Triangulation,
+// you may use this file under the same conditions.
+//
+// Author(s)    : Michael Kerber, Marc Glisse
+
+#include <CGAL/Delaunay_triangulation.h>
+#include <iterator>
+#include <vector>
+
+namespace Gudhi {
+
+/** \ingroup alpha_complex
+ *
+ * This function constructs the incremental Delaunay complex, as defined in \cite incremental-delaunay .
+ * Given a list of points (p1, ..., pn), the incremental Delaunay complex contains the Delaunay triangulation
+ * of (p1, ..., pk) for all k≤n. For any simplex s of the triangulation of (p1, ..., pk) that is not in the
+ * triangulation of (p1, ..., pn), the complex also contains a simplex formed with the same points as s plus the
+ * first pj that conflicts with s (is inside its circumsphere). For points in R^d, the complex usually has
+ * dimension d+1.
+ *
+ * This function calls `complex.insert_simplex_and_subfaces` on a set of simplices large enough that, with their
+ * faces, they define the incremental Delaunay complex. However, this set is not guaranteed to be minimal, there
+ * may be redundancies.
+ *
+ * No filtration values are computed here, those can be computed separately afterwards, for instance
+ * with `Gudhi::cech_complex::assign_MEB_filtration`.
```

Comment:
doxygen does not produce a link here. But this works:
```suggestion
 * with `Gudhi::cech_complex::assign_MEB_filtration()`.
```

Reply:
It works with doxygen 1.9.8. The CI has 1.9.1 which is starting to date a bit.

Reply:
Ok, keep it like that and I will update doxygen on my side before generating the official documentation

Reply:
I already changed it, I don't think it is worth changing back...